BackofficeBundle/Tests/Functional/Controller/EditNodeControllerTest.php

```diff
@@ -38,8 +38,9 @@ public function setUp()
      *
      * @dataProvider provideMetaAndNodeId
      */
-    public function testEditRootNode($expectedMeta, $newMeta, $nodeId)
+    public function testEditNode($expectedMeta, $newMeta, $nodeId)
     {
+        $this->markTestSkipped('The tested service has been desactivated');
```

Comment:
This skip is temporary, it was testing the NodeGroupRoleVoter, to allow the user to edit or not a specific page